build/python-client/Dockerfile

```diff
@@ -0,0 +1,42 @@
+ARG PYTHON_VERSION=3.10.4
+ARG GOGO_PROTOBUF_VERSION=1.3.2
```

Comment:
Can we get away with no needing the gogo proto stuff for the Python client?

Reply:
I think the only way to change the proto files.  

I see some small dependencies on it for submit.proto and event.proto.  

Our client seems to only use the submit.proto and event.proto but there are some usages of gogoproto.  

Basically, it seems you all use it for stdtime and nullable.  These are present in the user side of event and submit so I don't think we can remove without changing the API.  

All the event messages for submitting jobs seem to use the gogoproto.